stories/list/object/object-list.stories.js

```diff
@@ -0,0 +1,798 @@
+import '../../../dist/list.css';
+import '../../../dist/icon.css';
+import '../../../dist/checkbox.css';
+
+export default {
+    title: 'Components/List/Object',
+    parameters: {
+        description: `
+        Object list provides a structure for Object list item content.
+
+To create a object list add the \`fd-list--object\` modifier class to ul.
+(\`fd-list-optional\`) and (\`fd-list-optional__text\`) modifier class will help to add optional text information.
+(\`fd-list__object--top-row\`) and (\`fd-list__object--bottom-row\`) class will help to devide the list item into two parts.
+(\`fd-list__object--top-right\`) and (\`fd-list__object--top-left\`) modifiers will help to divide the top row into 2 column with ratio of 60% and 40%.
+Similary (\`fd-list__object--bottom-right\`) and (\`fd-list__object--bottom-left\`) modifiers will help to divide the bottom rows into 2 column with ratio of 50% and 50%.
+and to represent semantics for respective element append element-- \`neutral\`, \`positive\`, \`critical\`, \`negative\`
+and \`informative\` status.
+`,
+        tags: ['a11y', 'theme', 'development']
+    }
+};
+
+export const object = () => `
+<h4>Object List Item</h4>
+<ul class="fd-list fd-list--object" role="list">
```

Comment:
Would be a good idea to label all the unordered lists by something like the header.
```suggestion
<ul class="fd-list fd-list--object" role="list" aria-labelledby="objectListItemHeader">
```

Reply:
updated